src/main/kotlin/org/jetbrains/research/kotlin/mpp/inference/graph/Node.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.research.kotlin.mpp.inference.graph
+
+import NodeProto
+import TensorProto
+import org.jetbrains.research.kotlin.mpp.inference.operators.Operator
+import org.jetbrains.research.kotlin.mpp.inference.tensors.Tensor
+
+class Node(proto: NodeProto, val type: NodeType) {
+    val inputs: NodeIO = NodeIO()
```

Comment:
Why do we need inputs and outputs? Can't we get inputs in execute and return outputs?